source/help/messaging/formatting-text.rst

```diff
@@ -125,23 +125,44 @@ Renders as:
 Code Block
 ----------
 
-Create a code block by indenting each line by four spaces, or by placing ``````` on the line above and below your code.
+Creating a fixed width code block is recommended for pasting multi-line blocks of code or other text output that is easier to read with fixed width font alignment. Examples include block text snippets, ASCII tables, log files, etc.
 
-Example:
+Renders as:
 
 .. code-block:: none
 
+  this is my code
```

Comment:
Are you okay if we remove this example since we have two more examples following this

Reply:
yes; agreed, and done